test/expected/agg_bookends.out

```diff
@@ -3,7 +3,7 @@ SELECT create_hypertable('"btest"', 'time');
 NOTICE:  adding not-null constraint to column "time"
  create_hypertable 
 -------------------
- 
+ (1,public,btest)
```

Comment:
I'm confused, why do some return a result in the format of `(id,schema,table)` and other calls return an actual table?

Reply:
it returns table so output depends on wether you do
```SELECT function()
vs
SELECT * FROM function()```